src/application-insights/azext_applicationinsights/commands.py

```diff
@@ -88,3 +94,9 @@ def load_command_table(self, _):
 
     with self.command_group('monitor app-insights', query_sdk) as g:
         g.custom_command('query', 'execute_query')
+
+    with self.command_group('monitor app-insights component linked-storage', custom_command_type=component_linked_storage_accounts_custom_sdk) as g:
+        g.custom_show_command('show', 'get_component_linked_storage_account')
```

Comment:
Why do we need custom methods, but not call SDK directly?

Reply:
SDK is multi-api version. Since there are not enough tests for this extension, I want to keep it same as possible as I can.